project/Dependencies.scala

```diff
@@ -32,7 +32,7 @@ object Dependencies {
   val directoryWatcherVersion = "0.8.0+6-f651bd93"
   val mavenApiVersion = "3.6.1"
   val mavenAnnotationsVersion = "3.5"
-  val mavenScalaPluginVersion = "3.2.2"
+  val mavenScalaPluginVersion = "4.5.3"
```

Comment:
This version introduced the support for Scala 3 and Bloop is just reusing the plugin to read the data and generate configuration.